src/components/MyWorkplace/Pictures.tsx

```diff
@@ -8,11 +8,13 @@ import $api from "../../http";
 import {Button, Checkbox, FormControl, FormControlLabel} from "@material-ui/core";
 import Picture from './MyPicture';
 import MyPicture from "./MyPicture";
-import {StateOpenInterface} from "./WorkplaceTable";
+import {StateOpenInterface} from "./OneMasterOrder";
+
 
 interface PicturesProps {
     open: StateOpenInterface
     setOpen: Function
+    pictures: any
```

Comment:
any?

Reply:
corrected here https://github.com/mip92/clock_client/blob/6f71fd8cfc76c0f025fc36cc117fec7bde4ab733/src/components/MyWorkplace/Pictures.tsx. After merge it will be OrderPicture[]